Gem packaging via OS extension gem

x92499 opened the pull request:
David or Ryan,

I refactored this repo using the OS extension gem and believe I have completed everything up to releasing via github. Could you check this to make sure it looks ok before I complete the final steps. 

Thanks,
Karl

tijcolem commented on README.md:
```diff
@@ -1,3 +1,35 @@
+
+- [ ] Update measures to code standards
+
+# Releasing
```
You may want to add in a few more details regarding the gem release process. 
You can use the [openstudio-ext-gem ](https://github.com/NREL/openstudio-extension-gem) as a model. 
e.g:

# Releasing the gem

* Update CHANGELOG.md
* Run `rake rubocop:auto_correct`
* Update version in `/lib/openstudio/openstudio-load-flexibility-measures/version.rb`
* Create PR to master, after tests and reviews complete, then merge
* Locally - from the master branch, run `rake release`
* On GitHub, go to the releases page and update the latest release tag. Name it “Version x.y.z” and copy the CHANGELOG entry into the description box.

tijcolem commented on Rakefile:
```diff
@@ -0,0 +1,15 @@
+require 'bundler/gem_tasks'
```
looks good. I tested locally a couple of these built-in rake tasks and worked fine. 
`rake build`                                   
`rake clean`

tijcolem commented on openstudio-load-flexibility-measures.gemspec:
```diff
@@ -0,0 +1,32 @@
+
+  spec.add_dependency 'openstudio-extension', '~> 0.2.4'
+  spec.add_dependency 'openstudio-standards', '~> 0.2.11'
+end
```
Looks good. Since `openstudio-load-flexibility-measures-gem` is not available via rubygems, I did a simple test with a Gemfile that refs this branch and bundle installed. 

`$ echo "gem 'openstudio-load-flexibility-measures', git: 'https://github.com/NREL/openstudio-load-flexibility-measures-gem', branch: 'heine-gem'" > Gemfile && bundle install `

tijcolem commented:
I didn't comment on any of the rubocop changes.  If you want to use the same style guide that other openstudio ruby projects, you can use this file [rubocop.yml ](https://github.com/NREL/openstudio-extension-gem/blob/develop/.rubocop.yml )

tijcolem commented:
Lastly, this repo would probably benefit from adding in CI.  I can add a PR to this repo after this merge that does simple `bundle install && bundle exec rake` just to verify that the packaging/testing passes.

x92499 commented:
I added the .rubocop.yml and corrected all Errors. 6 Warnings remain re: method/classes exceeding specified number of lines. The gem readme.md could use additional detail, but individual measure documentation is thorough.
